### ISSUE TITLE
users permission question

### ISSUE THREAD

**Issue by LoloLaurentiu** [flyth/ts3soundbot#432]
Hello
just wanna ask if it is possible to add permission to play music for everybody without register. Maybe add permission to teamspeak guest group users.
2: where do I find the downloaded youtube music? (I run my bot on linux)
Thanks

**Comment by BlueFreakLP** [comment]
Hi @LoloLaurentiu ,
here is a instruction, how to install youtubedl. You must set the path to the executable in the sinusbot config and it should work.
https://rg3.github.io/youtube-dl/download.html

**Comment by LoloLaurentiu** [comment]
@BlueFreakLP yes I already did that. But I don't find the mp3 files. where is the location of the files when I upload them or when the bot downloads them from youtube

**Comment by BlueFreakLP** [comment]
Hi, they are stored directly to the ts3musicbot. On Thu, Apr 21, 2016 at 10:30 AM, Lolo < notifications@github.com [notifications@github.com] > wrote:
@BlueFreakLP [https://github.com/BlueFreakLP] yes I already did that. But I don't find the mp3 files. where is the location of the files when I upload them or when the bot downloads them from youtube
